Formula/fwdpp.rb

```diff
@@ -15,7 +14,7 @@ class Fwdpp < Formula
   end
 
   # build fails on Yosemite
-  depends_on "boost" => :build
+  depends_on "boost"
```

Comment:
```console
$ brew linkage fwdpp
…
Homebrew libraries:
  /usr/local/opt/boost/lib/libboost_program_options.dylib (boost)
```
For next time, could you please include a comment like 👆 in the PR to explain the change?

Reply:
Sorry, yeah.